EventsExpress/ClientApp/package.json

```diff
@@ -24,6 +24,7 @@
         "mdbreact": "4.19.0",
         "moment": "2.24.0",
         "moment-timezone": "0.5.26",
+        "oauth-sign": "0.9.0",
```

Comment:
Any reason for changing packages?

Reply:
I have added packages because of their use in code.

Reply:
My guessing is they are sub-dependencies of the twitter-login package, so we have not install them directly